### PR TITLE
#203 - changing all views and templates for multiple organizations

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: volontulo/models.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def __str__(self):
 class UserProfile(models.Model):
     u"""Model that handles users' profiles."""
     user = models.OneToOneField(User)
-    organization = models.ForeignKey(Organization, blank=True, null=True)
+    organizations = models.ManyToManyField(Organization)
```
Comment (filipgorczynski) [inline volontulo/models.py]: Maybe good idea would be to put here related_name='userprofiles' attribute?

File: volontulo/migrations/0003_auto_20150928_1908.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
```
Comment (filipgorczynski) [inline volontulo/migrations/0003_auto_20150928_1908.py]: Separate imports

File: volontulo/templates/offers/offer_form.html [hunk 3]
```diff
@@ -25,8 +25,10 @@
         <div class="form-group form-group-sm">
             <label class="col-xs-2 control-label" for="{{ offer_form.organization.id_for_label }}">Organizacja</label>
             <div class="col-xs-10">
-            <select value="{{ user.userprofile.organization.id }}" name="organization" class="form-control" id="{{ offer_form.organization.id_for_label }}">
-                <option value="{{ user.userprofile.organization.id }}">{{ user.userprofile.organization.name }}</option>
+            <select value="{{ user.userprofile.organizations.all.0.id }}" name="organization" class="form-control" id="{{ offer_form.organization.id_for_label }}">
```
Comment (filipgorczynski) [inline volontulo/templates/offers/offer_form.html:28]: How about handling this change within forms.py and CrateOfferForm?

File: volontulo/models.py (hunk 1 above)
Comment (magul) [inline volontulo/models.py]: done.

File: volontulo/templates/offers/offer_form.html (hunk 3 above)
Comment (magul) [inline volontulo/templates/offers/offer_form.html:28]: We spoke online and I drop this issue.

**Pull request merged**